ServerCore/ServerCore.csproj

```diff
@@ -31,6 +31,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="BuildBundlerMinifier" Version="2.9.406" />
+    <PackageReference Include="CsvHelper" Version="12.1.2" />
```

Comment:
Since this isn't a Microsoft package, we don't have a good way to ensure it's safe and will continue to work in the future. Could you please just build your own CSV strings (just make sure to put double quotes around values and escape quotes) or find a Microsoft-associated package?

Reply:
This package has 11.8 MILLION downloads on NuGet. There is no Microsoft associated CSV writing package (to my knowledge). Far more of a security risk that I would get the CSV writing rules incorrect; I have no desire to read and re-implement the CSV specification and all the corner cases. This is why NuGet exists.